pootle/apps/pootle_app/management/commands/refresh_scores.py

```diff
@@ -28,10 +29,51 @@ def add_arguments(self, parser):
             default=False,
             help='Reset all scores to zero',
         )
+        parser.add_argument(
+            '--user',
+            action='append',
+            dest='users',
+            help='User to refresh',
+        )
 
     def handle(self, **options):
+        self.stdout.write('Start running of refresh_scores command...')
+
+        User = get_user_model()
+        users = User.objects.all()
+        if options['users']:
+            users = users.filter(username__in=options['users'])
 
         if options['reset']:
-            User = get_user_model()
-            User.objects.all().update(score=0)
-            ScoreLog.objects.all().delete()
+            users.update(score=0)
+            scorelogs = ScoreLog.objects.all()
+            if options['users']:
+                scorelogs = scorelogs.filter(user__in=users)
+
+            scorelogs.delete()
+
+            if options['users']:
+                self.stdout.write('Specified users scores were reset to 0.')
+            else:
+                self.stdout.write('All user scores were reset to 0.')
+            return
+
+        start = datetime.datetime.now()
+        for user_pk, username in users.values_list("pk", "username"):
+            self.stdout.write("Process user %s..." % username)
```

Comment:
Nit: ongoing actions should use present continuous [as we do in other commands](https://github.com/translate/pootle/blob/master/pootle/apps/pootle_app/management/commands/__init__.py): _Processing_, _Setting_, _Running_...